python/tank/pipelineconfig_factory.py

```diff
@@ -538,10 +538,17 @@ def _get_pipeline_configs_for_path(path, data):
     for pc in data["pipeline_configurations"]:
 
         for s in storages:
-            # all pipeline configurations are associated
+
+            # all installed/classic pipeline configurations are associated
             # with a project which has a tank_name set
             project_name = pc["project.Project.tank_name"]
 
+            # if the project's tank_name is None, then this configuration is not
```

Comment:
similar: this is a little misleading i think. I think we should say something along the lines of

"this method is used to look up the appropriate configuration given a path on disk to an asset. Configurations that don't have a file system presence (templates/schema) can thus be safely ignored as a path will never be associated with such a configuration."

Reply:
addressed